CHANGELOG.md

```diff
@@ -53,6 +54,7 @@
 
 ## Breaking changes
 
+- Changed the the default `rtol` to `1e-4` in the (`IDAKLUSolver`). ([#4282](https://github.com/pybamm-team/PyBaMM/pull/4282))
```

Comment:
Is this breaking? It sounds like a pretty minor change

Reply:
This would also need to be under the unreleased section
```suggestion
```

Reply:
> Is this breaking? It sounds like a pretty minor change

It's a minor change, but it is breaking. If a user relies on the previous default `rtol`, then this will affect their results. Higher solver tolerances may be desirable in some cases (like optimization), so the prior default of `rtol=1e-6` is a reasonable choice for some situations